core/src/header.rs

```diff
@@ -1138,6 +1143,18 @@ impl fmt::Display for Length {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use crate::{dicom_value, DicomValue};
+
+    #[test]
+    fn to_clean_string() {
+        let p = dicom_value!(U16, [256, 0, 16]);
+        let val = DicomValue::new(p);
+        let element = DataElement::new(Tag(0x0028, 0x3002), VR::US, val);
+        match element.to_clean_str() {
+            Ok(value) => assert_eq!(value, "256\\0\\16"),
+            Err(_) => {}
+        }
```

Comment:
We don't want to ignore the error, because the operation should succeed. In this case we can just `unwrap` the result.

```suggestion
        assert_eq!(
            element.to_clean_str().unwrap(),
            "256\\0\\16",
        );
```